docs/dm/zh_CN/features/table-route.md

```diff
@@ -0,0 +1,34 @@
+table route rule
+===
+
+#### 功能介绍
+
+table route 提供将上游 MySQL/MariaDB 实例的某些表同步到下游指定表的功能。
+
+注意：
+- 不支持对同一个表设置多个不同的路由规则
+- schema 的匹配规则需要单独设置，用来同步 `create/drop schema xx`, 例如下面的 `rule-2`
+
+#### 参数配置
+
+```
+routes:                                             
+  rule-1:
+    schema-pattern: "test_*"                
```

Comment:
add introduce to `*` and `?` match rule. clarify it is not standard regular expression

Reply:
oh I found it in features/table-selector.md